dashboard/app/views/levels/_apps_dependencies.html.haml

```diff
@@ -24,7 +24,8 @@
 
 -# Droplet script dependencies (all editCode - regardless of editor)
 - if use_droplet
-  %script{src: asset_path('js/jsinterpreter/acorn_interpreter.js')}
+  %script{src: minifiable_asset_path('js/jsinterpreter/acorn.js')}
+  %script{src: minifiable_asset_path('js/jsinterpreter/interpreter.js')}
```

Comment:
Does this mean we're always depending on a dev-mode version of the interpreter now, or are you just building these differently?